code/modules/clothing/spacesuits/rig/rig.dm

```diff
@@ -558,7 +558,8 @@
 		ai_override_enabled = !ai_override_enabled
 		notify_ai("Synthetic suit control has been [ai_override_enabled ? "enabled" : "disabled"].")
 	else if(href_list["toggle_suit_lock"])
-		locked = !locked
+		if (locked != -1)
```

Comment:
User could use with some feedback why the button does nothing. Beeps in error etc.